.circleci/config.yml

```diff
@@ -10,114 +10,20 @@
-          name: 'Run Unit Tests'
-          command: |
-            cd presto-native-execution/_build/debug
-            ctest -j 8 -VV --output-on-failure --exclude-regex velox.*
-      - run:
-          name: 'Maven Install'
-          command: |
-            export MAVEN_OPTS="${MAVEN_INSTALL_OPTS}"
-            ./mvnw clean install ${MAVEN_FAST_INSTALL} -pl 'presto-native-execution' -am
-      - run:
-          name: 'Run e2e tests'
-          command: |
-            export PRESTO_SERVER_PATH="${HOME}/project/presto-native-execution/_build/debug/presto_cpp/main/presto_server"
-            export TEMP_PATH="/tmp"
-            mvn test ${MAVEN_TEST} -pl 'presto-native-execution' -DPRESTO_SERVER=${PRESTO_SERVER_PATH} -DDATA_DIR=${TEMP_PATH} -Duser.timezone=America/Bahia_Banderas -T1C
+setup: true
 
-  linux-parquet-S3-build:
-    executor: build
-    steps:
-      - checkout
-      - run:
-          name: "Update velox submodule"
-          command: |
-            cd presto-native-execution
-            make velox-submodule
-      - run:
-          name: "Install S3 adapter dependencies"
-          command: |
-            mkdir -p ${HOME}/adapter-deps/install
-            source /opt/rh/gcc-toolset-9/enable
-            set -xu
-            cd presto-native-execution
-            DEPENDENCY_DIR=${HOME}/adapter-deps PROMPT_ALWAYS_RESPOND=n ./velox/scripts/setup-adapters.sh
-      - run:
-          name: Build
-          command: |
-            source /opt/rh/gcc-toolset-9/enable
-            cd presto-native-execution
-            cmake -B _build/release -GNinja -DAWSSDK_ROOT_DIR=${HOME}/adapter-deps/install -DTREAT_WARNINGS_AS_ERRORS=1 -DENABLE_ALL_WARNINGS=1 -DCMAKE_BUILD_TYPE=Release -DPRESTO_ENABLE_PARQUET=ON -DPRESTO_ENABLE_S3=ON -DCMAKE_PREFIX_PATH=/usr/local -DCMAKE_CXX_COMPILER_LAUNCHER=ccache
-            ninja -C _build/release -j 8
+orbs:
+  path-filtering: circleci/path-filtering@0.1.5
 
-  format-check:
-    executor: check
-    steps:
-      - checkout
-      - run:
-          name: "Update velox"
-          command: |
-            cd presto-native-execution
-            make velox-submodule
-      - run:
-          name: Check formatting
-          command: |
-            cd presto-native-execution
-            make format-check
-
-  header-check:
-    executor: check
-    steps:
-      - checkout
-      - run:
-          name: "Update velox"
-          command: |
-            cd presto-native-execution
-            make velox-submodule
-      - run:
-          name: "Check license headers"
-          command: |
-            cd presto-native-execution
-            make header-check
+workflows:
+  always-run:
+    jobs:
+      - path-filtering/filter:
+          name: check-updated-files
+          # 3-column, whitespace-delimited mapping. One mapping per line
+          # <regex path-to-test> <parameter-to-set> <value-of-pipeline-parameter>
+          mapping: |
+            presto-native-execution/.* run_check_jobs true
```

Comment:
run_check_jobs is not precise because we have other jobs like s3 and macos in the future. how about changing it to 'run_native_specific_jobs' or something similar ?

Reply:
Renamed. Thanks.